tests/modules/sightings/resources/test_custom_fields.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+
+from tests.modules.sightings.resources import utils as sighting_utils
+from tests.extensions.edm import utils as edm_utils
+from tests import utils as test_utils
+
+
+CONFIG_PATH = '/api/v1/configuration/default'
```

Comment:
I don't think this is used

Reply:
curious the linter didnt pick this up.  :thinking: